README.md

```diff
@@ -582,6 +582,16 @@ This will be transmitted with each message. A count of unique users will appear
 
 You can now pass in empty strings (or false to `isAnonymous`) to reset the current user for login/logout scenarios.
 
+### Ending a session
+
+To end a user's current session:
+
+```javascript
+rg4js('endSession');
+```
+
+This will end the session for a user and start a new one.
```

Comment:
Perhaps having some documentation mentioning what happens to the any user information/tracking would be good as well. Just for extreme clarity :)

Reply:
"This will end the session for a user and start a new one. If the user has changed then they will be assigned to the new session, otherwise the new session will remain attached to the current user."

Something like this? 🤔

Reply:
Spoke in person - "This will end the session for a user and start a new one. The new session will remain attached to the current user."